### ISSUE TITLE
[BUG] "import" is not supported

### ISSUE THREAD

**Issue by jfcherng** [Sublime-Instincts/BetterTwig#13]
### Summary

Not sure why but I noticed the following line is commented out and thus `import` is not supported.

https://github.com/Sublime-Instincts/BetterTwig/blob/a005af78f06d17cd91e4800905e68e70a2dfecd7/resources/syntax/Twig.sublime-syntax#L58

### Problematic Twig template.

https://twig.symfony.com/doc/3.x/tags/import.html

```twig
{% import '_macro/handsontable.html.twig' as handsontable_macro %}
```

### Expected Behaviour

I simply un-comment it and get the following
![image](https://user-images.githubusercontent.com/6594915/97795802-e679df80-1c45-11eb-8529-d9cab1a23906.png)

### Actual Behaviour

![image](https://user-images.githubusercontent.com/6594915/97795809-fb567300-1c45-11eb-8e27-b03372c0c2e2.png)

### Environment

- Sublime Text build: 4090
- BetterTwig version: v1.0.3
- Twig version: v3.1.1

**Comment by Ultra-Instinct-05** [comment]
Thanks for the report. Currently, I am working on a bunch of improvements and this is on the list, now that I have a better idea of how to scope imports (At that time, I had some confusion so I left it commented out). This will be fixed in v1.0.4

**Comment by Ultra-Instinct-05** [comment]
This should be fixed as of v1.0.4
